### PR TITLE
Add update clause support for values function

### PR DESCRIPTION
**Pull request by passren**
This PR is to support get values from update and where clause.

Refer to issue Get values of WHERE, ORDER BY clauses #285

### CODE REVIEW

File: test/test_values.py [hunk 1]
```diff
@@ -93,3 +93,21 @@ def test_getting_values():
         "comment_parent": 0,
         "user_id": 0,
     }
+
+    parser = Parser(
+        "UPDATE `page_vote`"
+        "SET article_id = '442002', user_time = '27574631'"
+        "WHERE topic_id = '10000' AND section_id = '200000'"
+    )
```
Comment (macbre) [inline test/test_values.py:101]: Can we cover here passing numeric values as well? Would be nice to cover the `_format_value` funcion that you introduced.

**Comment by macbre** [comment]
@passren - thanks for the PR! I've left one comment above.

**Comment by macbre** [comment]
Can you also check the linting and formatting of the code?

Please run `make lint` and `black test/test_values.py`.

File: test/test_values.py (hunk 1 above)
Comment (passren) [reply test/test_values.py:101]: @macbre Sure. Will do that.

**Comment by passren** [comment]
@macbre A problem I faced is how to handle the same column including in SET and WHERE clause for UPDATE statement.

f.e. SQL: UPDATE table SET a=2, b=2 WHERE a=1 AND c=1
if call `parser.values_dict`, the value of 'a' column in SET will be overwrite by the value of 'a' in WHERE clause.

I propose 3 options to solve this:
1. return a list with ("column name", "column value") tuples
2. return a list with {"column name": "column value"} dicts
3. return a dict like `parser.columns_dict` returned:
```
{
        "update": ['value 1', 'value n'],
        "where": ['value 1', 'value n'],
}
```
For the statement of SELECT, return the same structure, like this:
```
{
        "insert": ['value 1', 'value n'],
}
```

All these options will change the manner of  `parser.values_dict` function. Welcome your comments.

**Comment by macbre** [comment]
I vote for number 3. All of these will break backward compatibility - but we can always bump the version to the next major :)

Thanks again for your contribution!

**Comment by passren** [comment]
> I vote for number 3 with each key having a list of (column, value) tuples (proposal no. 1).
> 
> Either way, all of these will break backward compatibility - but we can always bump the version to the next major :)
> 
> Thanks again for your contribution!

No problem! It's glad to do that since I used this fantastic module in my project.

**Comment by macbre** [comment]
> It's glad to do that since I used this fantastic module in my project.

Great to hear that!

**Comment by passren** [comment]
@macbre I have to close this PR since I have changed to pyparsing for SQL parser in my project since it's little bit hard to meet my requirement using sql-metadata.

**Pull request closed without merging**